hosts_example.yml

```diff
@@ -5,6 +5,13 @@ all:
     ansible_become: true
     ansible_ssh_private_key_file: /tmp/certs/ssh_priv.pem
 
+    #### Setting Proxy Environment variables ####
+    ## To set proxy env vars for the duration of playbook run, uncomment below block and set as necessary
+    # proxy_env:
```

Comment:
@JumaX do you think we shouldn't limit ourselves here to "only proxy env vars"? This could work for any env var, but it only is applied to playbooks that need proxy settings. I guess the question is- what other env vars might we want set during playbook runs

Reply:
@domenicbove That's a really great question.  My thoughts are that we should move forward with the proxy variable now, to solve the immediate problem, but we should add a jira to review these vars for each component going forward and add the most obvious ones which may need to be overridden by a user.